localinfo/forms.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, attrs=None, *args, **kwargs):
         default_options = {}
         options = kwargs.get('options', {})
         default_options.update(options)
-        for key, val in default_options.items():
+        for key, val in list(default_options.items()):
```

Comment:
entiendo que items retorna una lista, pq es necesario esto?

Reply:
A diferencia de python2, en python3 el método .items() retorna un iterador.

Reply:
pero en este caso debiese dar lo mismo porque el iterador debiese entregar una tupla key,val ¿qué error indica?

Reply:
Esto fue cambiado automáticamente por la librería 2to3. Estuve leyendo un poco del tema y es para mantener el comportamiento de lista en caso de que se requiera (pues en python2 retorna una lista), no obstante,  cuando se itera sobre un .items() existe el mismo comportamiento en python 2 y 3.

En este caso da lo mismo si es lista o no ya que no arroja ningún error y se usa para iterar nomás, es más eficiente no dejarlo como lista?
[Referencia](https://stackoverflow.com/questions/17695456/why-does-python-3-need-dict-items-to-be-wrapped-with-list)

Reply:
si, es mejor dejarlo como iterador, el criterio de elegir entre iterador o lista tiene que ver con el uso de memoria, la lista tiene todos los elementos en memoria mientras que un iterador mantiene solo el elemento de la iteración

Reply:
Ok